templates/index.html

```diff
@@ -347,7 +348,9 @@ <h3><span>Courses</span><small>Courses</small></h3>
 
                     </div>
                 </div>
-
+            </div>
+            <div class="center-item-outer-container">
+                <a href="https://www.khoury.northeastern.edu/information-for-overview/prospective-align-pillar/" class="btn custom-btn custom-btn-bg custom-btn-link learn-more-button" target="_blank">More Info</a>
```

Comment:
<img width="1200" alt="Screenshot 2023-09-20 at 3 16 09 PM" src="https://github.com/humans-of-align/humans-of-align-web/assets/62849837/742d7cf1-5ede-4174-af95-97bd60abfc84">